givitsite/friendreq/models.py

```diff
@@ -4,7 +4,7 @@
 
 class ItemRequest(models.Model):
     friend_id = models.IntegerField()
-    item = models.CharField(max_length = 40)
+    item = models.CharField(max_length = 40)git
```

Comment:
Good job. Almost there, though. Just remove the `git` from the end of this line.

Thanks and good job!